README.md

```diff
@@ -1,24 +1,99 @@
-# Examples for Using [PGO](https://github.com/CrunchyData/postgres-operator), the Postgres Operator from Crunchy Data
+This is a customized fork of [CrunchyData's Postgres Operator](https://github.com/CrunchyData/postgres-operator-examples/).
+
+Userdocs could be found [here](https://access.crunchydata.com/documentation/postgres-operator/latest/quickstart).
+
+Note: current installation does not expose posgtes outside the cluster (no ingress configuration for now).
+
+# Kubernetes deployment:
+## Install Postgres Operator (PRGO) and corresponding CRDs for k8s:
+
+`helm upgrade --install postgres-operator -n postgres-operator --create-namespace helm/install`
+
+When `--set singleNamespace=true`, PGO watches for and responds to PostgresClusters only in namespace, where it is installed. In this case, change namespace name in the following commands from `pgvector` to `postgres-operator` or other name you used to install operator itself.
+
+## Deploy Postgres (includes PGVector extension).
+### Minimal config:
+- 1 instance (no HA) with 2Gb RAM, 1vCPU, 2Gb storage (default StorageClass)
+- no pgBouncer, no TLS, no monitoring
+- pgBackRest stores backups in local PVCs
+- no default users or databases created
+
+`helm upgrade --install pgvector -n pgvector --create-namespace helm/postgres`
+
+### Default Apolo config:
+- 3 postgres replicas (high availability config, 1 master and 2 slaves controlled by Patroni). Resources: 2Gb RAM, 1vCPU, 20Gb storage (default StorageClass)
+- 2 pgBouncer replicas
+- Enabled Prometheus metrics exporter
+- 1 user is created with name `apolo`
+- 1 database is created with name `apolo`
```

Comment:
could we stick to `postgres` as a well-known default name?

Reply:
postgres user & databases are created too, as superusers, but I could use those names explicitly, wdyt?

Reply:
then perhaps we could simply offer the superuser credentials to the user? the users should be able to create as many other users and databases as they are pleased to.

Reply:
Adjusted the configuration, now the credentials are exposed for `postgres` user. End users will need to connect using this superuser to the cluster, create dedicated users & databases.

Reply:
I'll merge this MR, if any suggestions / questions -- will address in a follow-up